nixos/modules/installer/tools/nixos-install.sh

```diff
@@ -147,6 +147,14 @@ if [[ -z $system ]]; then
     outLink="$tmpdir/system"
     if [[ -z $flake ]]; then
         echo "building the configuration in $NIXOS_CONFIG..."
+
+        # We build the flake twice to populate the local nix store with
```

Comment:
Note that this is the *non-flake* path.

Reply:
I believe this is correct from the error messages I got when people are trying to install nixos for the first time.